sdk/servicebus/azure-servicebus/tests/test_message.py

```diff
@@ -304,7 +303,7 @@ def test_message_backcompat_receive_and_delete_databody(self, servicebus_namespa
             outgoing_message.message.reject()
         with pytest.raises(TypeError):
             outgoing_message.message.modify(True, True)
-        assert outgoing_message.message.state == uamqp.constants.MessageState.SendComplete
+        # assert outgoing_message.message.state == uamqp.constants.MessageState.SendComplete
```

Comment:
We probably should no longer make assertions using the .message attribute - as that was the leaked uamqp message (although once Swathi's switch PR is merged, we could probably uncomment these as it should be more-or-less backwards compatible.
But we should probably make a note to refactor these tests (if for no other reason than the test outputs are about to be flooded with deprecation warnings...)

Reply:
Since these are testing the LegacyMessage, I can refactor these tests in the switch PR so that we only call `.message` once.